src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard/MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard
+        key={movie.imdbId}
+        movie={movie}
+      />
```

Comment:
Don't break a line when you have only 2 props.

```suggestion
      <MovieCard key={movie.imdbId} movie={movie} />
```